helm/fdb-operator/templates/manager/deployment.yaml

```diff
@@ -20,6 +20,13 @@ spec:
       labels:
         {{- include "chart.labels" . | nindent 8 }}
     spec:
+      securityContext:
```

Comment:
Could you also update https://github.com/FoundationDB/fdb-kubernetes-operator/tree/master/config/samples/deployment they should always match

Reply:
Updated :)